.github/workflows/coding-standards.yml

```diff
@@ -173,3 +173,33 @@ jobs:
       SLACK_GHA_CANCELLED_WEBHOOK: ${{ secrets.SLACK_GHA_CANCELLED_WEBHOOK }}
       SLACK_GHA_FIXED_WEBHOOK: ${{ secrets.SLACK_GHA_FIXED_WEBHOOK }}
       SLACK_GHA_FAILURE_WEBHOOK: ${{ secrets.SLACK_GHA_FAILURE_WEBHOOK }}
+
+  failed-workflow:
+    name: Failed workflow tasks
+    runs-on: ubuntu-latest
+    needs: [ phpcs, jshint, slack-notifications ]
+    if: |
+      always() &&
```

Comment:
Why is the always present? Relic of testing?

Reply:
It's necessary because if a preceding job fails, all of the following jobs will not run, unless always() is there.